packages/vehicle-rental-overlay/package.json

```diff
@@ -18,15 +18,15 @@
     "url": "git+https://github.com/opentripplanner/otp-ui.git"
   },
   "dependencies": {
```

Comment:
A change in this package is causing at least this story to crash http://localhost:5555/?path=/story/vehiclerentaloverlay--rental-bicycles-visibility-controlled-by-knob

Reply:
Turns out it wasn't the package update! Looks like we forgot to update a reference to the new export names. See fe22f9936bb0c3705f5660a07ac42feb3f99c681